src/main/resources/application.yml

```diff
@@ -1,7 +1,9 @@
+logging:
+  level:
+    root:INFO
```

Comment:
Isn't log level handled by the environment variable `LOG_LEVEL`? i.e. locally we may want debug, whilst on Rebel1 we may want info, whilst on production we may only want errors and warnings. Hardcoding this makes this static.